api/src/main/java/org/openmrs/module/kenyaemr/reporting/builder/hiv/ETLLtfuReportBuilder.java

```diff
@@ -116,6 +117,7 @@ protected void addColumns(HybridReportDescriptor report, PatientDataSetDefinitio
 
         dsd.addColumn("Number of days late", daysMissed, paramMapping);
         dsd.addColumn("Program", new CalculationDataDefinition("Program", new PatientProgramEnrollmentCalculation()), "", new PatientProgramEnrollmentConverter());
+        dsd.addColumn("Physical Address", new PhysicalAddressDataDefinition(), "");
```

Comment:
You can re-use this for consistency. 
dsd.addColumn("Village_Estate_Landmark", new CalculationDataDefinition("Village/Estate/Landmark", new PersonAddressCalculation()), "", new RDQACalculationResultConverter());